app/src/main/java/in/avimarine/boatangels/activities/AddUserActivity.java

```diff
@@ -0,0 +1,77 @@
+package in.avimarine.boatangels.activities;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import butterknife.OnClick;
+import com.google.firebase.auth.FirebaseUser;
+import in.avimarine.boatangels.R;
+import in.avimarine.boatangels.db.FireBase;
+import in.avimarine.boatangels.db.iDb;
+import in.avimarine.boatangels.db.objects.User;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class AddUserActivity extends AppCompatActivity {
+
+  private String Name;
```

Comment:
Variables name should start with lowercase.

Reply:
i change to Lowercase.